clean.cmd

```diff
@@ -125,4 +139,11 @@ if NOT [%ERRORLEVEL%]==[0] (
   echo Command exited with ERRORLEVEL %ERRORLEVEL% >> %cleanlog%
   set clean_successful=false
 )
+exit /b
+
+:CleanEnvironment
+REM If VBCSCompiler.exe is running we need to kill it
+echo Stop VBCSCompiler.exe execution.
+echo. >> %cleanlog% && echo Stop VBCSCompiler.exe execution. >> %cleanlog% 
+for /f "tokens=2 delims=," %%F in ('tasklist /nh /fi "imagename eq VBCSCompiler.exe" /fo csv') do taskkill /f /PID %%~F >> %cleanlog%
 exit /b
```

Comment:
Yes I know, but that's how they checked it in to corefx, sigh. Trying to keep the diffs clean.